.gitignore

```diff
@@ -189,3 +189,6 @@ Third_Party/
 win64/
 dbg/
 rel/
+
+# Custom Project Configs
+.pre-commit-config.yaml
```

Comment:
We discussed offline that this should probably be removed for now as not relevant to this PR and not sure how widely useful it might be